sim/src/test/scala/bridges/BridgeSuite.scala

```diff
@@ -0,0 +1,92 @@
+//See LICENSE for license details.
+
+package firesim.bridges
+
+import java.io.File
+
+import org.scalatest.Suites
+import org.scalatest.matchers.should._
+
+import freechips.rocketchip.config.Parameters
+import freechips.rocketchip.config._
+
+abstract class BridgeSuite(
+  val targetName:  String, // See GeneratorUtils
+  targetConfigs:   String      = "NoConfig",
+  platformConfigs: String      = "HostDebugFeatures_DefaultF1Config",
+  tracelen:        Int         = 8,
+  simulationArgs:  Seq[String] = Seq(),
+) extends firesim.TestSuiteCommon
+    with Matchers {
+
+  val backendSimulator = "verilator"
+
+  val targetTuple    = s"$targetName-$targetConfigs-$platformConfigs"
+  val commonMakeArgs = Seq(
+    s"TARGET_PROJECT=bridges",
+    s"DESIGN=$targetName",
+    s"TARGET_CONFIG=${targetConfigs}",
+    s"PLATFORM_CONFIG=${platformConfigs}",
+  )
+
+  def run(
+    backend:  String,
+    debug:    Boolean      = false,
+    logFile:  Option[File] = None,
+    waveform: Option[File] = None,
+    args:     Seq[String]  = Nil,
+  ) = {
+    val makeArgs = Seq(
+      s"run-$backend%s".format(if (debug) "-debug" else ""),
+      "LOGFILE=%s".format(logFile.map(toStr).getOrElse("")),
+      "WAVEFORM=%s".format(waveform.map(toStr).getOrElse("")),
+      "ARGS=%s".format(args.mkString(" ")),
+    )
+    if (isCmdAvailable(backend)) {
+      make(makeArgs: _*)
+    } else 0
+  }
+
+  /** Runs MIDAS-level simulation on the design.
+    *
+    * @param b
+    *   Backend simulator: "verilator" or "vcs"
+    * @param debug
+    *   When true, captures waves from the simulation
+    * @param args
+    *   A seq of PlusArgs to pass to the simulator.
+    * @param shouldPass
+    *   When false, asserts the test returns a non-zero code
+    */
+  def runTest(b: String, debug: Boolean = false, args: Seq[String] = simulationArgs, shouldPass: Boolean = true) {
```

Comment:
Similarly, you want ` : Unit = ` here. 

I'll catch all of this when i get around to cleaning up some these tests.